releasenotes/notes/add_series_list_view-bf219022216fea6a.yaml

```diff
@@ -0,0 +1,11 @@
+---
+prelude: >
+    Some projects can have hundreds of patches actives at once, by giving the user
+    the ability to have an overview of all of the series makes knowing what's the
+    current state of a project much more simple.
```

Comment:
The `prelude` section is used to provide a high level overview of a release. Only a maintainer should create one of these. You can drop this.